apps/meteor/app/livechat/server/methods/takeInquiry.ts

```diff
@@ -54,6 +55,18 @@ export const takeInquiry = async (
 		throw new Error('error-mac-limit-reached');
 	}
 
+	const contactId = await (async () => {
+		if (!isSingleContactEnabled()) {
```

Comment:
isSingleContactEnabled has already been removed on the base branch, you can remove this check now.